model/model.cpp

```diff
@@ -0,0 +1,6 @@
+#include "model.h"
+
+int LumpedMass::stepForward(float dt, float heatAddition) {
+  temperature = temperature + heatAddition * dt / (m * cp);
+  return 0;
+}
```

Comment:
Core physics model implementation, which is in C++.  It should only depend on the model header file(s) and anything from the C++17 standard library.